spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/event/ClientApplicationHealthStatusChangedEvent.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2013-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.codecentric.boot.admin.event;
+
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.springframework.boot.actuate.health.Health;
+
+import de.codecentric.boot.admin.event.ClientApplicationStatusChangedEvent;
+import de.codecentric.boot.admin.model.Application;
+import de.codecentric.boot.admin.model.StatusInfo;
+
+/**
+ * This event gets emitted when an health event change is detected.
+ * 
+ * @author dbs
+ * @since 1.4.4
+ * @version 1.0.0
+ */
+public class ClientApplicationHealthStatusChangedEvent extends ClientApplicationStatusChangedEvent {
```

Comment:
Instead of adding a new EventType I suggest that a new property `details` of type `Serializable` is added to `ClientApplicationStatusChangedEvent`